CommonLib/DistantObject/GREYHostApplicationDistantObject.h

```diff
@@ -16,11 +16,11 @@
 
 #import <Foundation/Foundation.h>
 
-// Use modular imports for CocoaPods release.
-// #import <eDistantObject/EDOClientService.h>
-// End CocoaPods modular imports
-
+#if COCOAPODS
```

Comment:
Is this a define CocoaPods provides us?

Reply:
Yes

<img width="873" alt="image" src="https://user-images.githubusercontent.com/6798398/100711916-403f0800-33b2-11eb-943d-5ed9ad36974f.png">